lib/documentLoader.js

```diff
@@ -2,14 +2,38 @@
  * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
  */
 'use strict';
+/*eslint max-len: ["error", { "ignoreComments": true }]*/
 
 // load locally embedded contexts
 const contexts = require('./contexts');
 
+/**
+ * documentLoader is a set of methods used to dereference jsonld documents.
+ * Documents stored at urls are usually dereferenced via https requests
+ * while DIDs can be dereferenced by a variety of means.
```

Comment:
We should strike this sentence -- we don't mention what "DIDs" are and it doesn't make any sense why it's here to the reader. If the reader needs to know more about document loaders, we should add a reference to jsonld.js or the JSON-LD API spec as above -- otherwise they are just going to think that `documentLoader` is this little utility file that happens to be named after something from an entirely different library (jsonld.js).